src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,42 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```

Comment:
make sense to use useMemo here, because it's essential to performance

Reply:
used